src/book/01-team/05-robocup/03-debriefs.mdx

```diff
@@ -6,6 +6,126 @@ description: Debriefs and takeaways from our performances at RoboCup.
+
+#### Odometry
+
+- Pitch and roll were good, better than the previous year
+
+#### Vision
+
+- Fine tuning for vision worked well despite not having our own Theta camera
+
+#### Teamwork and Moral
+
+- Got robot ready for qualification very quickly
+- We all came to the games - had three different robot handlers for four games
+- Got lots of promotional footage of the robot
+- We made the quarter finals despite not having two whole set up days and the first competition day to test the system and networking
+- Within 30 minutes of getting our robots from the airport, the robots were walking and getting up
+- Good preparation before the competition was vital to the team being able to perform at all when the robots arrived at the end of the first competition day
+- Overall made good progress compared to RoboCup 2019 Sydney
+- On par or better than Team-KMUTT, UTRA, Hamburg Bit-Bots, Bold Hearts, ITAndroids, RoboFEI
+- Students at home were around to help with vision fine-tuning
+
+### What’s Next
+
+#### Networking
+
+- System configuration tool
+- Configurable headless mode, lightweight, if no system config tool due to it being tedious to set up networking
+- Team Robot communication – only one robot connects to GameController and then sends information to the other robots
+- Make GameController connection more robust
+- See how the small robots do team communication
+
+#### Hardware
+
+- Find a replacement for CM740’s quick so we can buy or make them before the end of the year.
+  - Redesign NUsense to only contain the core functionality – then additional functionality can be added with shields – modular system – modules that you can plug in
+- Get a decent battery powered soldering iron (TS100), a decent stand, sponge, and safety glasses with some back up wire and various connections
+- Better servos on the knees
+- Kneecaps to stop the knees overextending
+- Servo covers - couldn't get up from the back because of servos hitting each other
+- Improve cable management in the robot
+- Sort out a reel for longer ethernet cables
+- Look into purchasing small portable monitors for RoboCup
+- More out of the box / off the shelf solutions for hardware, such as a proper cooling system that we can buy off the shelf
+- Redesigning the back of the robot for easy access and able to give better padding protection
+
+#### Organisation
+
+- Target research / postgrad students - remove ‘maybe’ from do you plan on doing postgrad question
+- Attempt the git workshop again
+- Project management. Make a physical board
+- Create a culture of high performance
+- Keep momentum up, particularly post Robocup
+- Try and keep dev evenings early in the week (Monday, Tuesday) so people aren't busy with assignments
+- Make team desirable for academics
+- Come up with ideas on how to improve motivation
+- Keep dev evenings / time in the lab focused on development
+- Pack robots whole rather than disassembled - 2 in large case, one in square case, possibly carry-on
```

Comment:
I doubt you will get a robot into carry-on. They are heavy and bulky so they will exceed your carry-on allowance. Unless you plan on buying tickets for them so they can have a seat??? (do you need a passport for the robot to do that???)

Reply:
Bit-Bots were saying that they take their robots on carry on (despite going over the carry-on allowance)